ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml

```diff
@@ -15,88 +15,37 @@
   register: cygwin_installed
   tags: cygwin
 
-- name: Download Cygwin installer
+- name: Retrieve cygwin cache from ci.adoptopenjdk.net
   win_get_url:
-    url: https://cygwin.com/setup-x86_64.exe
-    dest: 'C:\temp\cygwin.exe'
-    force: no
-    checksum: 4dd4d4531e8e63ade849daaaf587ba1c1430368701772c8ee42a27f4e8c373e4
+    url: https://ci.adoptopenjdk.net/userContent/winansible/cygwin64.tar.gz
```

Comment:
Is it obvious where the tarball comes from and how it's created?

Reply:
I've been trying to keep a step by step in the linked issue (#1210) , which includes creating the cache- however, I'll write a comment explaining how it was made and link it above the task to make it more obvious